src/sentry/grouping/ingest/seer.py

```diff
@@ -163,7 +167,7 @@ def get_seer_similar_issues(
         "hash": event_hash,
         "project_id": event.project.id,
         "stacktrace": stacktrace_string,
```

Comment:
is it possible for this to affect the stacktrace as well?

Reply:
this likely is possible (though it maybe be properly filtered out already), but since we're only saving the vector embedding of the stacktrace in postgres, it's not a problem

Reply:
hm do you have the string handy by any chance? maybe we should check what happens when we call `encode` on it

Reply:
yeah I tried it out earlier - it'll just translate the byte string into raw (ex. `b'Title with null \x00, "\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00" is null'`)

Reply:
kk cool ty for verifying!!